src/msw/graphics.cpp

```diff
@@ -255,55 +255,66 @@ public :
     Matrix* m_matrix ;
 } ;
 
-class wxGDIPlusPenData : public wxGraphicsObjectRefData
+
+// Things that pens and brushes have in common
+class wxGDIPlusPenBrushBaseData : public wxGraphicsObjectRefData
+{
+public:
+    wxGDIPlusPenBrushBaseData(wxGraphicsRenderer* renderer);
+    ~wxGDIPlusPenBrushBaseData();
```

Comment:
Doesn't really change anything, but, for consistency:
```suggestion
    virtual ~wxGDIPlusPenBrushBaseData();
```

More importantly, I think it should be protected and not public as objects of this class are not supposed to be deleted, only `DecRef()`'d until they die on their own.